packages/opentelemetry-scope-base/README.md

```diff
@@ -6,6 +6,18 @@
 
 This package provides the ScopeManager interface (which is used by concrete implementations) and a no-op implementation (which is used internally when no scope propagation is defined). It's intended for use both on the server and in the browser.
 
+## What is a Scope Manager ?
+
+To understand why they exists, we'll need to understand how Javascript works: when you make native function call (networks, setInterval etc) you generally call C++ code that will later callback your own code.
+
+A common issue when tracing a request in javascript is to link the function that have made the native call to the callback that the native code called when the response is there. Imagine you want to track for which user you made the request, you need some sort of "context/scope aware storage".
+
+ScopeManager's aim to offer exactly that, it's API offer to store an object in the current scope (`with()`) and if needed, `bind()` to a specific function call to find it back when the callback fire, which can later get retrieved using `active()`.
+
+This package only include the interface and a Noop implementation, for more information please see the [async-hooks based ScopeManager][ah-scope-manager] for NodeJS.
+
+Note: You will sometimes read about **context** management, which is sometimes also used for talking about **scope** management. However in OpenTelemetry, **context** was already heavily used as a concept in the Trace API so we named this package with **scope** to avoid confusion.
```

Comment:
```suggestion
Note: You will sometimes read about **context** management, which is sometimes also used for talking about **scope** management. However, in OpenTelemetry **context** was already heavily used as a concept in the Trace API so we named this package with **scope** to avoid confusion.
```